Command/DumpCommand.php

```diff
@@ -45,4 +50,25 @@ private function parseOptions(InputInterface $input)
 
         return $options;
     }
+
+    /**
+     * @throws \LogicException
+     */
+    protected function getContainer(): ContainerInterface
```

Comment:
Could maybe DI the service within the constructor, although could have issues with older versions of Symfony that I'm not aware of.

Reply:
Yeah maybe but I'm also not sure about older versions. Didn't want to introduce a potentially breaking change.